Assets/HoloToolkit/UX/Resources/Materials/3DTextSelawik.mat

```diff
@@ -0,0 +1,136 @@
+%YAML 1.1
```

Comment:
Duplicate of https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/master/Assets/HoloToolkit/UX/Materials/3DTextSelawik.mat

Reply:
@keveleigh Removed duplicated 3DTextSelawik material.

Reply:
Did any scenes use the old material and need to be updated? Also, there was a second duplicate 3DTextSelawik material as well. There's one in HoloToolkit/UX/Materials, and this PR is adding another in HoloToolkit/UX/Resources/Materials

Reply:
@keveleigh Deleted the one under Resources and verified the scenes. Thanks!